week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,21 @@
 */
 
 function calculateGrade(mark) {
-
+  if(mark <= 49){
+    return "F";
+  }
+  else if (mark >= 90){
+    return "A";
+  }
+  else if (mark >= 70 ){
+    return "B";
+  }
+  else if (mark >= 55){
```

Comment:
Don't use the specific grades provided to create your conditions, use the ranges provided in the instructions - see lines 4 to 8 of this file.
The specific grades (55, 90, 49 etc) are there to test your function.

Reply:
@LucyMac Noted will make necessary update